kinde_sdk/auth/oauth.py

```
@@ -95,6 +103,19 @@ def __init__(
         self.proxy = None
         self.proxy_headers = None
 
+        # Initialize route protection (optional)
+        self._route_protection = None
+        if route_protection_file and ROUTE_PROTECTION_AVAILABLE:
+            try:
+                self._route_protection = RouteProtectionEngine(route_protection_file)
+                self._logger.info(f"Route protection enabled with config: {route_protection_file}")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)